src/usr/ipmiext/ipmidcmi.C

```diff
@@ -52,6 +159,27 @@ namespace SENSOR
 
         errlHndl_t l_err = NULL;
 
+        // Power Management support check
+        bool support;
+        l_err = getPowerManagementSupportStatus(support);
+        if (l_err != NULL)
+        {
+            // Since the Power Management support information isn`t received,
+            // ignore this error and still try to read the power limit. If the
+            // Power Management is really unsupported, the DCMI Get Power Limit
+            // command will return error code in l_cc
+            TRACFCOMP(g_trac_ipmi,
+                      "Failed to check Power Management support");
```

Comment:
Suggested text change "Failed to determine if the BMC supports Power Management"

Reply:
Done